internal/wire/record.go

```diff
@@ -36,8 +38,87 @@ type Record struct {
 	BodyB   []byte
 }
 
+// LoadDataDocuments is like LoadRecords, but instead of loading raw records,
+// it extracts valid data documents.
+func LoadDataDocuments(dir string, limit int) ([]*types.Document, error) {
+	files, err := findBinFiles(dir)
+	if err != nil {
+		return nil, err
+	}
+
+	var docs []*types.Document
+
+	// We can't tell whether a record have any data documents or
+	// not before parsing the bin file.
+	// Parsing all files and then using some shuffling is slow,
+	// so we're using a random window to collect the results.
```

Comment:
Random window seems fine :+1: Given we were not getting any before, having any randomization seems positive!